docs/articles/documentation/reference/test-api/testcontroller/presskey.md

```diff
@@ -7,7 +7,7 @@ redirect_from:
 ---
 # t.pressKey Method
 
-Presses the specified keyboard keys.
+Presses the specified keyboard keys. Does not require a target element.
```

Comment:
I'm not sure "Presses" is the best word to use, since an automated test doesn't exactly "press" the keyboard keys. Could we say the following? (Let me know what you think.)

`Selects the specified keyboard keys. Does not require a target element.`

Reply:
This method is specifically tailored to emulate key presses. Sure there is no actual keyboard or buttons to press, but the effect is exactly the same. `Selects` doesn't make sense from a technical standpoint.

Reply:
Okay, your version is fine then.